algorithms/Java/trees/Left-Leaf-Sum.java

```diff
@@ -0,0 +1,93 @@
+package Trees.Left_Leaf_Sum;
+/*
+Input:-
+ 				 3
+ 			 /	 \
+ 			9     20
+ 		        /  \
+           15    8
```

Comment:
Rectify the indentation of the tree
Suggestion - Draw the same tree which you have considered as the input to this program
This will help the viewer understand it more clearly